README.md

```
@@ -45,13 +45,17 @@ If you want to contribute, you’ll find the workflow approachable, the structur
 1. **Scaffold:**  
    `./tools/start_here.sh newmod`
 2. **Develop & Document:**  
-   Edit `.sh`, `.conf`, `.md` in `staging/`
+   Edit `.sh`, `.conf` in `staging/` optional 250x250 logo image optional 
 3. **Validate:**  
    `./tools/10_validate_module.sh`
 4. **Promote:**  
-   Maintainers move working modules to `src/`
-5. **Consolidate:**  
-   Prepare release by flattening to `lib/`
+  ` ./tools/20_promote_module.sh`
```

markdownlint-cli2 (0.17.2)

Spaces inside code span elements

(MD038, no-space-in-code)